packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.test.js

```diff
@@ -0,0 +1,60 @@
+import test from 'ava'
+import getForwardedActions from './get-forwarded-actions'
+import { from } from 'rxjs'
+
+test('should receive and filter the forwardedActions registry', async (t) =>{
+  t.plan(2)
+
+  const forwardedActionsObservable = from([
```

Comment:
I would make some of these different, and test against a unique one. In theory, the observable should never have duplicates in it like this right? Also, ensuring access to more than just the first value would probably be good. As it is it's just passing as soon as the first value is read in so having multiples doesn't really make sense.